.github/workflows/intel-19.yml

```diff
@@ -0,0 +1,27 @@
+name: Intel 19.1 
+
+on:
+  push:
+    branches: [master, static]
+  pull_request:
+    branches: [master, static]
+
+jobs:
+  build:
+    runs-on: self-hosted
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: environment
+      run: source /opt/intel/bin/compilervars.sh intel64 
+    - name: cmake
+      run: |
+        mkdir -p build
+        cd build
+        cmake  -DCMAKE_C_COMPILER=icc -DCMAKE_CXX_COMPILER=icpc -DCMAKE_BUILD_TYPE=Debug ..
```

Comment:
Why debug out of curiosity?

Reply:
I did look at the oneAPI solution, and the self-hosted option gives us a bit more control on the target architecture and version etc. I will modify this to use Github runner and re-submit. 

I retained the flags in the GCC worklflow :)

Reply:
You retained the flags from the gcc debug workflow ;-).  I test debug and release with gcc because there's bugs in gcc debug compilation.